user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/List.md

```diff
@@ -1,18 +1,20 @@
 ---
-uid: DashboardTreeFeed
+uid: DashboardList
 ---
 
-# Tree feed
+# List
 
-This feed component allows the user to select one or more items in a tree view. The selectable items can be based on any data feed.
+This component allows the user to select one or more items in a list. The selectable items can be based on any data feed.
+
+![List](~/user-guide/images/List.gif)<br>*List component in DataMiner 10.4.6*
 
 To configure the component:
 
-1. Apply the necessary data feeds. See [Applying a data feed](xref:Apply_Data_Feed).
+1. Apply the necessary data. See [Adding data to a component](xref:Adding_data_to_component).
 
    > [!NOTE]
    >
-   > - From DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->, when you apply a single query data source or *Tables* component data source, individual rows from that query or table are listed instead of the query or table itself. Additionally, if you want to use this tree feed component as a data feed, the entry will be listed as *Tables* or *Query rows* in the *Data* pane, depending on your DataMiner version<!--RN 41075-->. When you apply multiple query data sources or *Table* component data sources, the queries or tables themselves are listed as data. Prior to DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4, when you apply a single query data feed, the query itself is listed as data. If you use this tree feed component as a data feed, the entry will be listed as *Queries* in the *Data* pane.
+   > - From DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->, when you apply a single query data source or *Tables* data source, individual rows from that query or table are listed instead of the query or table itself. Additionally, if you want to use this list component as a data feed, the entry will be listed as *Tables* or *Query rows* in the *Data* pane (depending on your DataMiner version<!--RN 41075-->). When you apply multiple query data sources or *Tables* data sources, the queries or tables themselves are listed as data. Prior to DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4, when you apply a single query data source, the query itself is listed as data. If you use this list component as data, the entry will be listed as *Queries* in the *Data* pane.
```

Comment:
if you want to use this list component as a data feed => if you want to use this list component as data

Reply:
```suggestion
   > - From DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4 onwards<!--RN 38811-->, when you apply a single query data source or *Tables* data source, individual rows from that query or table are listed instead of the query or table itself. Additionally, if you want to use this list component as data, the entry will be listed as *Tables* or *Query rows* in the *Data* pane (depending on your DataMiner version<!--RN 41075-->). When you apply multiple query data sources or *Tables* data sources, the queries or tables themselves are listed as data. Prior to DataMiner 10.3.0 [CU13]/10.4.0 [CU1]/10.4.4, when you apply a single query data source, the query itself is listed as data. If you use this list component as data, the entry will be listed as *Queries* in the *Data* pane.
```